src/TestExplorer/TestDiscovery.ts

```diff
@@ -89,10 +89,6 @@ export function updateTests(
     testItems.forEach(testItem => {
         upsertTestItem(testController, testItem);
     });
-
-    testController.items.forEach(item => {
-        testController.items.add(item);
-    });
```

Comment:
Surprised I missed this. What was this meant to be doing?

Reply:
I left this in accidentally, it was effectively a no-op